docs/integrations/azure_pipelines.md

```diff
@@ -176,7 +183,7 @@ documentation for more detail.
 
 Choose the Docker image tag to match your comfort level with image dependencies. `latest` is a "rolling" tag that
 will point to the image created for the latest released `phylum-ci` Python package. A particular version tag
-(e.g., `0.21.0-CLIv4.0.1`) is created for each release of the `phylum-ci` Python package and _should_ not change
+(e.g., `0.42.4-CLIv6.1.2`) is created for each release of the `phylum-ci` Python package and _should_ not change
```

Comment:
Are you actually continuously updating this? That just seems unmaintainable.

I feel like `0.1.2-CLIv3.4.5` or something similarly obviously "wrong" would avoid needless churn like that. But I also think having an outdated valid value would be fine, after all it's an **example**.

Reply:
I saw this version is also used in some actual code files. How is one intended to update this reliably without missing one of these?

Reply:
> Are you actually continuously updating this? That just seems unmaintainable.

No. I only update these opportunistically, when other nearby changes are made.

> I feel like `0.1.2-CLIv3.4.5` or something similarly obviously "wrong" would avoid needless churn like that. But I also think having an outdated valid value would be fine, after all it's an **example**.

Agreed...the outdated valid value works just fine and I have no urge to update it except when I am already updating nearby text.